### ISSUE TITLE
Maya 2020

### ISSUE THREAD

**Issue by tomasjanik** [Latios96/claudius#2]
Hello, your project looks amazing! I would like use Claudius in Maya 2020 How can I do it? After editing .bak file form 2018 in to 2020 (haha) is impossible check Claudius in Maya plugin manager. I really would like experiment and work with that plugin.
Thanks for response. 
Tom

**Comment by AlanEdd** [comment]
Yes would be super to use this in Maya, it's really a missing piece of the jig saw in Maya.
